week-5/DaInglichTim/queries/README.md

```diff
@@ -0,0 +1,77 @@
+# Solutionary
+
+## Show the total amount of sales in January 2009
+
+```sql
+SELECT COUNT(id_transaction)
+FROM transactions;
+```
```

Comment:
I think this query answers the **total number of sales** and not the **total amout of sales**

Reply:
I may have misunderstood the question, I'll fix it